src/components/teasers/teaser.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { Teaser } from './teaser';
+import ExampleImage from '../../../blog-posts/images/angular.png';
+import styled from 'styled-components';
+import { Illustration, IllustrationSize } from '../illustration/illustration';
+import { ILLUSTRATION_NAMES } from '../illustration/illustration-set';
+
+export default {
+  component: Teaser,
+  title: 'Components/Teaser',
+} as ComponentMeta<typeof Teaser>;
+
+const Template: ComponentStory<typeof Teaser> = (args) => <Teaser {...args} />;
+
+export const Regular = Template.bind({});
```

Comment:
The teasers looks unnatural wide in the previews because they naturally spread. I would have expected a story where you showcase the natural habitat: A grid that holds multiple instances of a teaser. Something like the special showcase story like I did for the illustrations to show all illustrations.